### ISSUE TITLE
How to create a podspec with two platforms

### ISSUE THREAD

**Issue by erikdoe** [CocoaPods/CocoaPods#4075]
How do you specify two platforms in a podspec file? The reason I'm asking is that I'm trying to create a podspec for iOS and OSX. I have read #3925 and have no problem with being explicit. But how?

The documentation isn't exactly clear about this. See: https://guides.cocoapods.org/syntax/podspec.html#platform

I have tried to specify the platform multiple times, but it seems that the later one only overrides the previous one. I've also tried to specify a hash, i.e.

    spec.platform = { :ios => nil, :osx => nil }

That doesn't work either. Any ideas?

By the way, there seems to be a trick that involves specifying multiple deployment targets: https://github.com/erikdoe/ocmock/commit/7662f9e1e3c73b17592949262eaa9f7943f9b305

If I go down that route, how do I specify different deployment targets based on architecture? The project requires iOS 7.0 for 64-bit builds but runs on iOS 5.0 for all others...

**Comment by segiddins** [comment]
```ruby
s.ios.minimum_deployment_target = '7.0'
s.osx.minimum_deployment_target = '10.9'
```

In the future, please try and ask usage questions such as this on a venue like Stack Overflow, we try and keep issues reserved for bugs in CocoaPods.

**Comment by erikdoe** [comment]
Apologies. I should have been clearer. I do know about the way to achieve this using the deployment targets. This is what was done in the commit I referenced in the original post.

What I am after is a way to specify multiple platforms _without_ specifying a deployment target. Similar to what was discussed in #3925.

If this is not possible consider this a feature request. ;-) If it is possible then the documentation on the link I provided needs to be updated.
